prime-angular-frontend/src/app/modules/gis-enrolment/pages/ldap-user-page/ldap-user-page.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, HostListener, OnInit } from '@angular/core';
 import { FormControl } from '@angular/forms';
```

Comment:
I know you didn't add this, it's not used anymore.

Reply:
nice catch, removed